deploy/minikube/release_sanity_test.go

```diff
@@ -70,10 +72,43 @@ func TestReleasesJson(t *testing.T) {
 			actualSha, err := getShaFromURL(getDownloadURL(r.Name, platform))
 			if err != nil {
 				t.Errorf("Error calcuating SHA for %s-%s. Error: %s", r.Name, platform, err)
+				continue
 			}
 			if actualSha != sha {
 				t.Errorf("ERROR: SHA does not match for version %s, platform %s. Expected %s, got %s.", r.Name, platform, sha, actualSha)
+				continue
```

Comment:
Yeah sorry, to be clear we'll still get an error for each release that fails, but only one error per release instead of several.

If we don't have this continue, we'd see something like:

Error calculating URL!
Error downloading SHA!
Error, SHA does not match!

when really all we want to see is the first error.